<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="labelHeader">
       <property name="font">
        <font>
         <family>Impact</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Project Distro</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>1281</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelFooter">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>8</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Aaryan Khadka | Abhinab Badal | Binayak Dhungana | Prinshav Kandel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>291</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="Mean">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mean</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelMean">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="Median">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Median</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelMedian">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="Mode">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelMode">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>291</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="SD">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Standard Deviation</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelSD">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="Var">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Variance</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelVar">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="IQR">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Interquartile Range</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="labelIQR">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>681</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonNew">
       <property name="text">
        <string>New File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCalc">
       <property name="text">
        <string>Calculate Graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>20</width>
      <height>611</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QcustomPlot" name="customPlot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>681</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>20</width>
      <height>611</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LabelA">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelAminus">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>570</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBplus">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBminus">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCplus">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>570</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelC">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCminus">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabelD">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>701</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A-:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B+:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B-:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C+:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C-:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C-:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>550</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcustomPlot</class>
   <extends>QGraphicsView</extends>
   <header>qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

